src/components/Share/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import Style from './style.scss'
+
+export default class Share extends Component {
+
+  render() {
+    let {surahId, name} = this.props.surah;
+    let url = encodeURIComponent(`http://quran.com/${surahId}`);
+
+    let facebookUl = `https://www.facebook.com/sharer/sharer.php?u=${url}`;
+    let twitterUrl = `https://twitter.com/intent/tweet?url=${url}&text=Surat ${name.simple}`;
+
+    return (
+      <div className={Style.share}>
+
+        <i onClick={()=> this.onClickPopup(facebookUl, "Facebook")}
+           className={Style.facebook}
+           data-metrics-event-name="Share:Facebook"
+           title="Share on Facebook"></i>
+
+        <i onClick={()=> this.onClickPopup(twitterUrl, "Twitter")}
+           className={Style.twitter}
+           data-metrics-event-name="Share:Twitter"
+           title="Share on Twitter"></i>
+
+      </div>
+    )
+  }
+
+  onClickPopup(url, title)  {
```

Comment:
hmm, i would have thought eslint would complain about this being under `render`...